pymc3/sampling.py

```diff
@@ -1207,6 +1207,66 @@ def sample_ppc_w(traces, samples=None, models=None, weights=None,
     return {k: np.asarray(v) for k, v in ppc.items()}
 
 
+def sample_prior(samples=500, model=None, vars=None, size=None,
+                   random_seed=None, progressbar=True):
+    """Generate samples from the prior of a model.
```

Comment:
I would add a bit more description of why this is useful. It's really the prior predictive we're sampling from here.

Reply:
If its helpful -- I think one use case for this function is to generate a unique starting point for each chain, when multiple are required, like in this case #2856